projects/TransFusion/configs/transfusion_nusc_pillar_L.py

```diff
@@ -0,0 +1,397 @@
+_base_ = ['../../../configs/_base_/default_runtime.py']
+custom_imports = dict(
+    imports=['projects.TransFusion.transfusion'], allow_failed_imports=False)
+point_cloud_range = [-51.2, -51.2, -5.0, 51.2, 51.2, 3.0]
```

Comment:
Please rename this file as `transfusion_lidar_pillar02_second_secfpn_8xb2-cyclic-20e_nus-3d.py`

Reply:
renamed config at https://github.com/open-mmlab/mmdetection3d/pull/2547/commits/615cb1d6fa1bd3f1dc3876a5120cf2b8ffdf534d.